juju-tutorials/how-to-fixing-attach-volume-to-a-k8s-instance.md

```diff
@@ -0,0 +1,336 @@
+[comment]: <> (How To Perform Fixing Attach Volume to Instance)
+
+## pre-install
```

Comment:
Requirements? Have a look to https://github.com/canonical/doc-docs/pull/3/files for similar terminology (like the "Requirements" section).

Reply:
Thanks, I divided it into "Introduction" & "Requirements".